clms-angular/src/app/modules/dashboard/pages/dashboard-page/dashboard-page.component.ts

```diff
@@ -32,10 +33,19 @@ export class DashboardPageComponent implements OnInit {
   fetchAssignments() {
     const courseIds = this.myCourses.map(course => course.id);
     courseIds.forEach(courseId => {
-      this.httpClient.get<AssignmentProjection[]>(`/api/courses/${courseId}/assignments`).subscribe((data) =>
+      this.httpClient.get<AssignmentProjection[]>(`/api/courses/${courseId}/assignments`).pipe(map(assignments => {
+        return assignments.map(assignment => {
+          if (typeof assignment.startDate === 'string') {
+            assignment.startDate = new Date(assignment.startDate);
+          }
+          if (typeof assignment.dueDate === 'string') {
+            assignment.dueDate = new Date(assignment.dueDate);
+          }
+          return assignment;
+        });
+      })).subscribe((data) =>
```

Comment:
String's aren't converted to Dates at runtime

Reply:
made a pipe checkout branch: SA-StringToDate

Reply:
ignore this. I didn't see you already did it